spec/controllers/idv/hybrid_mobile/socure/document_capture_controller_spec.rb

```diff
@@ -175,6 +176,74 @@
         expect(response).to be_not_found
       end
     end
+
+    context 'when socure error encountered' do
+      let(:fake_socure_endpoint) { 'https://fake-socure.com/' }
```

Comment:
```suggestion
      let(:fake_socure_endpoint) { 'https://fake-socure.test/' }
```
👀 https://github.com/18F/identity-idp/pull/11289#discussion_r1787887246